db/db.go

```diff
@@ -4,9 +4,15 @@ import (
 	"context"
 )
 
+//Storer type interface
 type Storer interface {
 	ListUsers(context.Context) ([]User, error)
+	GetUser(context.Context, int) (User, error)
```

Comment:
when ever there is built in datatype in method use below conversion

GetUser(ctx context.Context, id int) (user User, err error)